### DISCUSSION TITLE
__patch: 和 patch: 有什么区别？

### DISCUSSION THREAD

**Discussion by jack6th** [rime/home#1006] [General]
在某个schema.yaml 里看到有用到 __patch: 
custom.yaml里有patch:
有什么不一样？

**Comment by lotem** [comment]
詳見：
https://github.com/rime/home/wiki/Configuration

`__patch` 是Rime的YAML編譯器指令。
`a.custom.yaml:/patch` 只是一個YAML map的名字。

Rime的YAML配置文件有一項隱含的內容（編譯步驟）是從`*.custom.yaml`文件讀取補丁，相當於寫：
```yaml
# a.schema.yaml
__patch: a.custom.yaml:/patch
```
